.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,60 @@
+
+name: 🐞 Bug
+description: Report an issue to help improve the project.
+title: "[BUG] <title>"
+labels: ["bug"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A brief description of the question or issue, also include what you tried and what didn't work
+    validations:
+      required: true
+  - type: textarea
+    id: Reproduce
+    attributes:
+      label: To Reproduce
+      description: Steps to reproduce this bug
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: Please add screenshots if applicable
+    validations:
+      required: false
+  - type: textarea
+    id: extrainfo
+    attributes:
+      label: Additional information
+      description: Is there anything else we should know about this bug? (like linux distribution(version + arch), python version and etc)   
+    validations:
+      required: false
+  - type: dropdown
+    id: component
+    attributes:
+      label: Where did you see this issue?
+      description: In which pbench component this issue was found?
+      options:
+        - Agent
+        - Server
+        - Other
+    validations:
+      required: true
+  - type: input
+    id: component-version
+    attributes:
+      label: In which version? 
+      description: In which version this issue was found?
+      placeholder: v0.71
```

Comment:
Placeholder's are just a hint, even if the text is getting rendered there but the value of that textbox will be blank/empty .
Since i have added an validations `required: true` that makes the input-box an mandatory field and it cannot be kept blank.
One can just enter "-" or NA if unaware of the version.

Reply:
Since the user is required to type something into this field, I don't think it's a good idea to put `v0.71` there.  For one thing, it gives the appearance that the field is already filled in (albeit, in a lighter shade of gray, but, if that's the value that the user wants, why would s/he think to re-type it?).  For another, this is a value which will go stale over time.

I suggest either leaving the field without a placeholder or supplying a placeholder which is generic and obviously needs to be replaced (like `vX.XX`).

BTW, it feels slightly "unpolished" to have an auto-disappearing placeholder for this field while the `title` field comes with a pre-supplied value which has to be manually edited.

Reply:
> I suggest either leaving the field without a placeholder or supplying a placeholder which is generic and obviously needs to be replaced (like vX.XX).

vX.XX seems to be good option inside placeholder.

> BTW, it feels slightly "unpolished" to have an auto-disappearing placeholder for this field while the title field comes with a pre-supplied value which has to be manually edited.

Yes i was thinking about the same, will make use of placeholders for title as well

Reply:
Yeah, placeholders make more sense for these than default text, because we can require the user to at least deliberately enter *something* rather than leaving a likely useless default value.

Reply:
We can `Additional information` Textarea instead.